### PR TITLE
Added camera stuff

### PR DESCRIPTION
**Pull request by n00kii**

### CODE REVIEW

**Review by yashpatel1392 — APPROVED** [review]
.

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -19,4 +19,5 @@
     public static int MAX_COLOR_ROTATIONS = 3;
     public static String SPIN_COLOR = "";
     public static boolean IS_SPIN_COMPLETE = false;
+    public static boolean IS_CAPTURE_COMPLETE = true;
```
Comment (GregoryLeMasurier) [inline src/main/java/frc/robot/Constants.java]: Is this used anywhere? I couldn't find any references.

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -20,6 +20,10 @@
 
 
 import frc.robot.Constants;
+import frc.robot.commands.CaptureColor;
+import frc.robot.subsystems.ColorSensor;
+
+import frc.robot.subsystems.*;
```
Comment (GregoryLeMasurier) [inline src/main/java/frc/robot/Robot.java]: I don't think we need this import.

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -20,6 +20,10 @@
 
 
 import frc.robot.Constants;
+import frc.robot.commands.CaptureColor;
+import frc.robot.subsystems.ColorSensor;
```
Comment (GregoryLeMasurier) [inline src/main/java/frc/robot/Robot.java]: I don't think we need this import.

File: src/main/java/frc/robot/subsystems/ColorSensor.java [hunk 4]
```diff
@@ -8,8 +8,12 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardComponent;
```
Comment (GregoryLeMasurier) [inline src/main/java/frc/robot/subsystems/ColorSensor.java]: Do we need the shuffleboard imports?

File: src/main/java/frc/robot/Robot.java (hunk 3 above)
Comment (n00kii) [reply src/main/java/frc/robot/Robot.java]: Delete

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (n00kii) [reply src/main/java/frc/robot/Constants.java]: Delete

File: src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (n00kii) [reply src/main/java/frc/robot/Robot.java]: Delete

File: src/main/java/frc/robot/subsystems/ColorSensor.java (hunk 4 above)
Comment (n00kii) [reply src/main/java/frc/robot/subsystems/ColorSensor.java]: Was experimenting to try to get shuffleboard or smartdashboard to display colors but that failed so d e l e t e

**Pull request closed without merging**